manifests/host.pp

```diff
@@ -7,6 +7,8 @@
   Hash $options     = {},
   String $comment   ='',
   Boolean $ignored  = false,
+  Optional[Integer] $default_lease_time = undef,
```

Comment:
Please enforce the minimal integer length. I assume 0 or 1 is the lowest value? For 0, the datatype would be:
```suggestion
  Optional[Integer[0]] $default_lease_time = undef,
```

Reply:
@bastelfreak the man page does not say what the minimum values are.  I don't feel that enforcing correctness in this way is appropriate.  If they haven't stated minimums, perhaps there's a reason (e.g., it changes over versions or they want to keep options open where a negative value may indicate some special case).  If you really want to ensure that kind of safety, it would make much more sense to create the proposed config file on the side and then delegate the validation using `dhcpd -t -cf TEMPFILE` , but I'm not volunteering for that, I simply don't have the time.